src/main/kotlin/race/model/Winner.kt

```diff
@@ -0,0 +1,8 @@
+package race.model
+
+class Winner(private val cars: List<Car>) {
+    fun getWinner(): String = cars.maxOf { it.position }.let {
+        cars.filter { car -> car.position == it }.joinToString { it.name }
+    }
```

Comment:
스타일의 차이긴한데,
개인적으로는 let 과같은 범위지정함수의 경우, 불필요하게 쓰는걸 지양하는 편입니다!
(참고만해주세요!)
```suggestion
    fun getWinner(): String {
        val winnerPosition = cars.maxOf { it.position }
        return cars.filter { car -> car.position == winnerPosition }
            .joinToString { it.name }
```

Reply:
저는 let 함수를 좋아해서요.  마지막 return block(this) 좋아해서요 
참고하겠습니다.